### PR TITLE
[OneExplorer] Add rename shortcut 

### PR DESCRIPTION
**Pull request by dayo09**
This commit supports shortcut for renaming.

ONE-vscode-DCO-1.0-Signed-off-by: Dayoung Lee <dayoung.lee@samsung.com>

### CODE REVIEW

File: src/OneExplorer/OneExplorer.ts [hunk 1]
```diff
@@ -442,6 +442,7 @@ export class OneNode extends vscode.TreeItem {
     super(node.name, collapsibleState);
 
     this.id = node.id;
+    this.node = node;
```
Comment (jyoungyun) [inline src/OneExplorer/OneExplorer.ts:445]: Do you need node variable?

**Review by jyoungyun — DISMISSED** [review]
LGTM

**Comment by dayo09** [comment]
@Samsung/one-vscode_committers PTAL :-D

File: src/OneExplorer/OneExplorer.ts [hunk 2]
```diff
@@ -587,9 +587,22 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<Node> {
       vscode.commands.registerCommand("one.explorer.delete", (node: Node) =>
         provider.delete(node)
       ),
-      vscode.commands.registerCommand("one.explorer.rename", (node: Node) =>
-        provider.rename(node)
-      ),
+      vscode.commands.registerCommand("one.explorer.rename", async () => {
+        if (provider.getSelectedCfg()?.length !== 1) {
+          // Delete is only supported for single selection
```
Comment (seanshpark) [inline src/OneExplorer/OneExplorer.ts:592]: Q) command is 'rename` but comment is 'Delete'. is this OK?

Comment (dayo09) [inline src/OneExplorer/OneExplorer.ts:592]:
```suggestion
          // Rename is only supported for single selection
```

**Review by seanshpark — DISMISSED** [review]
LGTM

**Review by jyoungyun — APPROVED** [review]
LGTM

**Comment by dayo09** [comment]
I changed a bit to resolve conflict and put related changes to #1593

File: src/OneExplorer/OneExplorer.ts [hunk 3]
```diff
@@ -614,6 +614,24 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<Node> {
...
+          } else {
+            const node = provider.getSelectedCfg()![0];
+            Logger.info("OneExplorer", "Shortcut", `Rename ${node.uri.fsPath}`);
+
+            await provider.rename(node);
+            // TODO: improve refresh performance
+            provider.refresh(node.parent);
+          }
+        }
+      ),
```
Comment (jyoungyun) [inline src/OneExplorer/OneExplorer.ts:634]: Is it right indentation?

Comment (dayo09) [reply src/OneExplorer/OneExplorer.ts:634]: Yes, it's formatted with the format.sh

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**